noxfile.py

```diff
@@ -96,7 +102,9 @@ def showcase_library(
         opts = "--python_gapic_opt="
         opts += ",".join(other_opts + (f"{template_opt}",))
         cmd_tup = (
-            f"protoc",
+            "python",
+            "-m",
+            "grpc_tools.protoc",
```

Comment:
What does this accomplish? Does it handle protoc versioning better?

Reply:
`protoc` doesn't have to be installed separately. Not directly related to this PR though! https://grpc.io/docs/languages/python/quickstart/#grpc-tools

Reply:
Cool, good to know.